src/Function/Bundles.agda

```diff
@@ -188,14 +188,54 @@ module _ (From : Setoid a ℓ₁) (To : Setoid b ℓ₂) where
       }
 
     open IsLeftInverse isLeftInverse public
-      using (module Eq₁; module Eq₂; strictlyInverseˡ)
+      using (module Eq₁; module Eq₂; strictlyInverseˡ; isSurjection)
 
     equivalence : Equivalence
     equivalence = record
       { to-cong   = to-cong
       ; from-cong = from-cong
       }
 
+    isSplitSurjection : IsSplitSurjection to
+    isSplitSurjection = record { from = from ; isLeftInverse = isLeftInverse }
+
+    surjection : Surjection
+    surjection = record
+      { to = to
+      ; cong = to-cong
+      ; surjective = λ y → from y , inverseˡ
+      }
+
+  -- A left inverse is also known as a “split surjection”.
```

Comment:
I don't know whether it's right to call an inhabitant of `LeftInverse` “a left inverse”.

Reply:
No, but such usage seems to have been present since the old hierarchy (in `Function.Inverse`)...